cypress/integration/test/activities.spec.js

```diff
@@ -0,0 +1,101 @@
+const url = 'http://localhost:3000/activities';
+const id = 1;
+const noId = 99999;
+
+const activity = {
+    name: 'TEST Name',
+    content: 'TEST Content'
+}
+
+describe('Endpoint /activities', () => {
+
+    it('Get all activities', () => {
+        cy.request('GET', url)
+            .should((response) => {
+                expect('Content-Type', /json/)
+                expect(response.status).to.eq(200)
+                expect(response.body).to.not.be.null
```

Comment:
Artemio esta muy bien en general. Cambiaria el expect(X).to.not.be.null por un expext(X).to.deep.equal(X) es decir, no testear por lo que no es, sino por lo que se espera. Esto aplica para todos los tests

Reply:
Hola, Maxi. Ya realicé los cambios en los response. Aguardo tu comentarios